ecommerce/migrations/0007_unitofmeasure_productprice.py

```
@@ -0,0 +1,47 @@
+# Generated by Django 4.2.23 on 2025-08-27 04:21
+
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ecommerce', '0006_cart'),
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)